pages/02-01-up-and-running.md

```diff
@@ -22,15 +22,93 @@ Along with installing LÖVE, you will need a text editor for creating Lua files
 
 When you launch LÖVE, (see instructions below on how to do that) you will be greeted with a friendly graphic and the text "NO GAME", meaning you are running the engine but didn't give it a game to load.
 
-### macOS and other posix systems
+## macOS
+
+Once you have downloaded the LÖVE binary for macOS (64-bit zipped), proceed to the "Downloads" folder and unzip the archive. You should now see an application called "love".
+
+![macOS-love](/images/02-01-macos-folder.png)
+
+macOS might show you a warning modal, because you are trying to open an application by an unverified developer. If so, right click on the application and choose "open" and "open" again in the following dialog. You should now be greeted by the no-game screen.
+
+### Starting your game
+
+The simplest way to start a LÖVE game is to drag the whole folder containing the game's source files (not just the main.lua file!) onto the application file.
```

Comment:
At this point the read doesn't have any games unless we tell them where to download one. Perhaps this section should be moved after they create their "hello world" game closer to the bottom of the section.

Reply:
This section of dragging and dropping to start the game works for both macOS and Windows so perhaps it would be easiest to just move it under the "Run the game" section as a "Drag and drop method".

For now I don't think different pages is necessary for each operating system.

Reply:
I'm having trouble with separating the "install instructions" and the running instructions since they are tightly coupled. After all it only makes sense to explain the terminal setup with an alias if I also show how to use it (which essentially is already showing how to start a game).

Reply:
Oh wait nvm... please check the latest version I just pushed!